ew/cmd/race/racecmds.py

```diff
@@ -169,12 +169,55 @@ async def rattle(cmd):
                 " a little bit too hard. Oof ouch owie.",
                 " so viciously they actually get offended.",
                 " in an attempt to socialize, but they don't think you should.",
+                ", inviting them to have a bad time." #blame loom
             ]
-            response = "You rattle your bones at {}{}".format(cmd.mentions[0].display_name, random.choice(responses))
+            responses = [
```

Comment:
there's another list called responses just above. you should make sure you're not overwriting that with this